src/DankCord/Objects.py

```diff
@@ -183,16 +183,32 @@ def __init__(self, data: dict) -> None:
         self.dropdowns: list = [
             item for component in self.components for item in component.components if isinstance(item, Dropdown)
         ]
+        try:
+            self.nonce: str = data["nonce"]
+        except:
+            # Unless used wrongly, it just means the object was used on a MESSAGE_UPDATE data, those don't have nonce
```

Comment:
It shouldn't be used wrongly. If it is, it should raise an exception rather than silently ignore it to cause pain elsewhere.

Reply:
I figured a user wouldn't try to use the Message function on something anyway, and if that's the case, the lib would use it correctly